libnetdata/ebpf/ebpf.c

```diff
@@ -249,66 +256,186 @@ int has_condition_to_run(int version)
 
 //----------------------------------------------------------------------------------------------------------------------
 
-char *ebpf_kernel_suffix(int version, int isrh)
+/**
+ * Kernel Name
+ *
+ * Select kernel name used by eBPF programs
+ *
+ * Netdata delivers for users eBPF programs with specific suffixes that represent the kernels they were
+ * compiled, when we load the eBPF program, the suffix must be the nereast possible of the kernel running.
+ *
+ * @param selector select the kernel version.
+ *
+ * @return It returns the string to load kernel.
+ */
+static char *ebpf_select_kernel_name(uint32_t selector)
```

Comment:
What's the point of using this function and magic numbers in the next one? It's very difficult to correlate them. Aren't macros with string literals better?

Reply:
I agree, I pushed two commits, in the first I removed the integers with a enum, and the last I added a macro with value that remembers the enum values.